src/main/java/yapper/ui/Ui.java

```diff
@@ -0,0 +1,98 @@
+package yapper.ui;
```

Comment:
Everything else seems good. 
-adhered to the naming conventions
-overall tidy layout 
-consistent ordering of import statements